website/docs/docs/deploy/ci-jobs.md

```diff
@@ -79,6 +79,26 @@ To make CI job creation easier, many options on the **CI job** page are set to d
 
 </Tabs>
 
+
+## Trigger a CI job with the API
+
+If you're not using dbt Cloud’s native Git integration with [GitHub](/docs/cloud/git/connect-github), [GitLab](/docs/cloud/git/connect-gitlab), or [Azure DevOps](/docs/cloud/git/connect-azure-devops), you can use the [Administrative API](/docs/dbt-cloud-apis/admin-cloud-api) to trigger a CI job to run. However, dbt Cloud will not automatically delete the temporary schema for you. This is because automatic deletion relies on incoming webhooks from Git providers, which is only available through the native integrations.
```

Comment:
Are there scenarios where they may have Git integration and still want to use the API to trigger CI jobs? Just wondering if the first part is necessary
```suggestion
The [Administrative API](/docs/dbt-cloud-apis/admin-cloud-api) can trigger a CI job to run if you're not using native Git integration. However, dbt Cloud will not automatically delete the temporary schema for you. This is because automatic deletion relies on incoming webhooks from Git providers.
```

Reply:
good question, @matthewshaver !

@knartovich : is what matt brings up a potential use case? or is this only for non-native git integrations?

Reply:
i am not sure how common of a use case that is, but it's definitely possible to leverage the API for non-native integrations as well. however, if they have webhooks set up, that could cause multiple runs being kicked off for the same PR commit

Reply:
thank you everyone! 

after reading these comments, my take-away is that this might not be a use case we should be documenting, @matthewshaver . i'll go ahead and leave this as is. we can update later if this changes.